crates/swc/tests/errors/lints/no-alert/alert-ident-hoisting/.swcrc

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Why is this removed?

Reply:
Oh... It's should be reversed back. I was remove this test because it covers by no-use-before-define